eth/stagedsync/stage_execute.go

```diff
@@ -953,3 +957,19 @@ func PruneExecutionStage(s *PruneState, tx kv.RwTx, cfg ExecuteBlockCfg, ctx con
 	}
 	return nil
 }
+
+func tryUpdateForkVersion(cfg *ExecuteBlockCfg, hermezDb *hermez_db.HermezDb) error {
```

Comment:
I know I didn't get around to that, but can this function be in the `stage_execute_zkevm.go`?

Reply:
Done